lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -46,6 +48,17 @@ export default ({
     },
     cache,
     plugins: [
+      refs
+        ? new VirtualModulePlugin({
+            [path.resolve(path.join(configDir, `generated-refs.js`))]: `
+              import { addons } from '@storybook/addons';
+
+              addons.setConfig({
+                refs: ${JSON.stringify(refs)},
+              });
+            `,
+          })
+        : null,
```

Comment:
We might want to eventually generalize passing static data from `main.js` to the manager (and the preview for that matter), but this is 👍 for now.

Reply:
agreed